extra/python/rx.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env python
+# Description: Simple netmap rx example
+# Author: Faisal T. Abu-Nimeh (abunimeh@)
+
+import netmap
+import select
+
+IF_NAME = 'em0'
+
+d=netmap.NetmapDesc('netmap:' + IF_NAME)
+rxr = d.receive_rings[0] # single ring example
+# use poll
+poller = select.poll()
+poller.register(d.getfd(), select.POLLIN)
+
+while True:
+    ready_list = poller.poll() # wait for data
+    if len(ready_list) == 0:
+        print("Timeout occurred")
+        break;
+
+    if not rxr.empty(): # only process nonempty ring
+        i = rxr.cur
+
+        flen = rxr.slots[i].len
+        if flen > rxr.nr_buf_size: continue # drop large packets
+        fbuf = rxr.slots[i].buf[:flen] # this will fail if flen > nr_buf_size, i.e., nr_buf_size must be >= mtu
```

Comment:
Yes em.

The slots immediately after the 1st one contain the remaining of the packet. However, each slot takes only 2048. If I recall correctly, If I don't drop the packet, I was able to recover it from the next 4 slots.